frontend/src/components/Shared/VideoPlayer/VideoPlayer.jsx

```
@@ -1,229 +1,432 @@
+        aria-label="Video controls"
+        tabIndex="0"
+      >
+        <input
+          type="range"
+          min="0"
+          max={duration || 1}
+          step="0.01"
+          value={currentTime}
+          onChange={(e) => {
+            if (videoRef.current) {
+              videoRef.current.currentTime = parseFloat(e.target.value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)